docs/en/nodejs-course.md

```diff
@@ -0,0 +1,25 @@
+## RS School NodeJS course
+
+The course is dedicated for frontend developers who would like to get familiar with NodeJS and the server side of web application development.
+
+## Prerequisites
+
+Solid knowledge of JavaScript including ES6.
+
+## Course duration
+
+The course duration is 5 weeks. For self-study, you'll need at least 8 hrs per week.
+
+## Course content
+
+During the course progress you'll need to create a CLI application for NodeJS and also a REST server.
```

Comment:
During the course you'll need to create a CLI application for NodeJS and a REST server.